modules/fundamental/src/sbom/service/sbom.rs

```diff
@@ -590,7 +591,7 @@ struct PackageCatcher {
     purls: Vec<Value>,
     cpes: Value,
     relationship: Option<Relationship>,
-    licenses: Option<String>,
+    licenses: Vec<Value>,
```

Comment:
I think it should be possible to use an actual struct for this already: https://www.sea-ql.org/SeaORM/docs/generate-entity/entity-structure/#column-type a bit at the end of the section: `FromJsonQueryResult`